pkg/cmd/cli/describe/describer.go

```diff
@@ -336,3 +341,70 @@ func (d *PolicyBindingDescriber) Describe(namespace, name string) (string, error
 		return nil
 	})
 }
+
+// TemplateDescriber generates information about a template
+type TemplateDescriber struct {
```

Comment:
I think there's one or two more refactors coming to describers - after that I think so.